notebooks/utils.py

```diff
@@ -72,10 +118,27 @@ class ZoningInfo:
     zone_class: str = ""
     D: bool = False
     height_district: str = ""
-    overlay: typing.Optional[typing.List[str]] = None
-
+    overlay: typing.Optional[typing.List[str]] = ""
+    invalid_zone: str = ""
+    invalid_height: str = ""
+    specific_plan: str = ""
 
     def __init__(self, zoning_string: str):
+        try:
+            self.general_parser(zoning_string)
+        except ValueError:
+            pass
+        try:

```

Comment:
I don't think this quite works for when the `general_parser` succeeds, since this `try` block is entered unconditionally